### PR TITLE
Mongodb connection through Express server API

### PR DESCRIPTION
**Pull request by EliranGooner**

### CODE REVIEW

File: my-app/backend/mongodbClient.js
Comment (ZvikaNaorCohen) [inline my-app/backend/mongodbClient.js:1]: So here the new code is basicly the commented code, but un-commented. Plus the tables we were discussing about in the zoom meeting, right?

Comment (ZvikaNaorCohen) [reply my-app/backend/mongodbClient.js:1]: (I'm referring to lines 19 to 220)

Comment (EliranGooner) [reply my-app/backend/mongodbClient.js:1]: There are some differences to match our needs but they're not drastic

Comment (ZvikaNaorCohen) [reply my-app/backend/mongodbClient.js:1]: LGTM. IMO Can merge.

**Pull request merged**